include/dr/detail/communicator.hpp

```diff
@@ -227,7 +228,7 @@ class rma_window {
   void create(communicator comm, void *data, std::size_t size) {
     local_data_ = data;
     communicator_ = comm;
-    DRLOG("win create:: size: {} data:{}", size, data);
+    DRLOG("win create:: size: {} data:{}", (MPI_Aint)size, data);
```

Comment:
1. no c-style casts
2. why the cast is needed?

Reply:
It was necessary during debug (usigned/signed conversion), now removed,